### DISCUSSION TITLE
aws-node pods call deprecated APIs removed in 1.22

### DISCUSSION THREAD

**Discussion by tomsucho** [aws/amazon-vpc-cni-k8s#2077] [Q&A] [answered]
I was doing some audit log filtering for EKS as part of readiness checks for 1.22 upgrade and I can see aws-node pods send API requsts to a number of APIs that are removed from 1.22:
```- admissionregistration.k8s.io/v1beta1  
- apiextensions.k8s.io/v1beta1			
- apiregistration.k8s.io/v1beta1		
- authentication.k8s.io/v1beta1			
- authorization.k8s.io/v1beta1			
- certificates.k8s.io/v1beta1			
- coordination.k8s.io/v1beta1			
- extensions/v1beta1					
- networking.k8s.io/v1beta1				
- rbac.authorization.k8s.io/v1beta1		
- scheduling.k8s.io/v1beta1				
- storage.k8s.io/v1beta1				
```
it seems that this only starts on pod startup and then is not repeated, so I hope this won't be an issue post upgrade to 1.22?

**Comment by jayanthvn** [comment, marked as answer]
This is handled post 1.22 release. Please ref - https://github.com/aws/amazon-vpc-cni-k8s/issues/1074
